res/values/colors.xml

```diff
@@ -24,6 +24,8 @@
     <color name="owncloud_blue_bright">#00ddff</color>
 
     <color name="list_item_lastmod_and_filesize_text">#989898</color>
+    <color name="actionModeBackground">#757575</color>
+    <color name="actionModeStatusBarBackground">#616161</color>
```

Comment:
Maybe, these color must be on setup.xml, to configure them taking into account the actionbar color.
Could you change the names of them using "_" instead of capital letters as in the rest of colors?